src/Tutor.Core/DomainModel/Feedback/IFeedbackRepository.cs

```diff
@@ -9,5 +9,7 @@ namespace Tutor.Core.DomainModel.Feedback
     public interface IFeedbackRepository
     { 
         void SaveEmotionsFeedback (EmotionsFeedback emotionsFeedback);
+
+        void SaveCrowdReFeedback (CrowdReFeedback crowdReFeedback);
```

Comment:
Ensure you change all the related names once you select a more meaningful option.